explorer/src/pages/TransactionDetailsPage.tsx

```diff
@@ -82,7 +93,36 @@ function StatusCard({ signature }: Props) {
     }
   }, [signature, clusterStatus]); // eslint-disable-line react-hooks/exhaustive-deps
 
-  if (!status || status.status === FetchStatus.Fetching) {
+  React.useEffect(() => {
+    return () => {
+      if (autoRefresh.timeout) {
+        clearTimeout(autoRefresh.timeout);
+      }
+    };
+  }, [autoRefresh]);
+
+  React.useEffect(() => {
+    if (
+      !autoRefresh.isAutoRefresh &&
+      status?.data?.info?.confirmations !== "max"
```

Comment:
I think you'll want to check if `status?.data?.info` has been initialized first. Otherwise this will always be true when the page is loading